.github/workflows/image-build-alpine-binary.yml

```diff
@@ -0,0 +1,82 @@
+##
+## Build the Docker Alpine image from the pre-built downloaded binary asset
+##
+
+name: Build Alpine Binary Image
+
+env:
+  REPO_OWNER: stacks-network
+
+# Only run when:
+#   - manually triggered via the ci.yml workflow
+
+on:
+  workflow_call:
+    inputs:
+      tag:
+        required: true
+        type: string
+        description: "semver tag for alpine images"
+      docker_platforms:
+        required: true
+        description: "Arch to buid alpine images"
+        type: string
+    secrets:
+      DOCKERHUB_USERNAME:
+        required: true
+      DOCKERHUB_PASSWORD:
+        required: true
+
+jobs:
+  image:
+    if: ${{ inputs.tag != '' }}
+    name: Build Image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the latest code
+        id: git_checkout
+        uses: actions/checkout@v3
+      - name: Set Vars
+        id: set_vars
+        run: |
+          echo "GITHUB_SHA_SHORT=${GITHUB_SHA::7}" >> $GITHUB_ENV
+          echo "GITHUB_REF_SHORT=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Set up QEMU
+        id: docker_qemu
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        id: docker_buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Docker Metadata
+        id: docker_metadata
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            ${{ env.REPO_OWNER }}/${{ github.event.repository.name }}
+          tags: |
+            type=raw,value=latest,enable=${{ inputs.tag != '' && (github.ref == format('refs/heads/{0}', github.event.repository.default_branch) )}}
+            type=raw,value=${{ inputs.tag }},enable=${{ inputs.tag != '' }}
+            type=ref,enable=true,priority=600,event=tag
```

Comment:
Not sure `priority=600` is doing anything here since the other tags don't specify their priority.

Reply:
https://github.com/wileyj/stacks-blockchain/commit/d8ce51e8d9a4990728bc43acf5e48c80a8079103